### ISSUE TITLE
[UX2.0] omp parcel fails with Invalid Json Payload Input

### ISSUE THREAD

**Issue by tzarski0** [CiscoDevNet/terraform-provider-sdwan#224]
When you configure below omp parcel:
```
resource "sdwan_system_omp_profile_parcel" "system_omp_profile_parcel" {
  advertise_ipv4_bgp          = false
  advertise_ipv4_connected    = true
  advertise_ipv6_bgp_variable = "{{omp_advertise_ipv6_bgp}}"
  description                 = "basic omp"
  ecmp_limit_variable         = "{{omp_ecmp_limit}}"
  feature_profile_id          = "76b21a53-b780-4e87-9652-71447378eb57"
  graceful_restart            = true
  graceful_restart_timer      = 86400
  id                          = (known after apply)
  name                        = "omp"
  version                     = (known after apply)
}
```

It fails with:
```
│ Error: Client Error
│ 
│   with module.sdwan.sdwan_system_omp_profile_parcel.system_omp_profile_parcel["system1-omp"],
│   on ../terraform-sdwan-nac-sdwan/sdwan_profile_parcels.tf line 297, in resource "sdwan_system_omp_profile_parcel" "system_omp_profile_parcel":
│  297: resource "sdwan_system_omp_profile_parcel" "system_omp_profile_parcel" {
│ 
│ Failed to configure object (POST), got error: HTTP Request failed: StatusCode 400, {"error":{"message":"Invalid Json
│ Payload Input","code":"SCHVALID0001","details":"{\"Validation Errors\":{\"Required But Missing
│ Attributes\":[\"data.overlayAs\"]}}","type":"error"}}
```

This worked 1-2 weeks ago and now it fails.

**Comment by danischm** [comment]
https://github.com/CiscoDevNet/terraform-provider-sdwan/commit/189046b85dde395d3088bf191360b86235eefcb6

**Comment by tzarski0** [comment]
Fixed in v0.3.8 release.
